benchmarks/time_forest_partition.cxx

```diff
@@ -463,7 +472,7 @@ main (int argc, char *argv[])
       || T <= 0 || test_tet + test_linear_cylinder + test_occ_cylinder > 1
       || (cmesh_level >= 0 && (!test_linear_cylinder && !test_occ_cylinder))
       || ((mshfileprefix != NULL || cmeshfileprefix != NULL)
-          && (test_linear_cylinder || test_occ_cylinder || test_tet))) {
+          && (test_linear_cylinder || test_occ_cylinder || test_tet || use_occ))) {
```

Comment:
This change results in not being able to specify a `msh` file and using the `use_occ` option at the same time. The program is only executed, if the `if` statement is false